recipes/qhull/all/patches/fix-cmake-8.0.x.patch

```diff
@@ -0,0 +1,141 @@
+     qhull_TARGETS_TEST   # Unused
+         user_eg user_eg2 user_eg3 user_egp testqset testqset_r
+@@ -375,7 +374,8 @@ add_library(${qhull_SHAREDR} SHARED
+         src/libqhull_r/qhull_r-exports.def)
+ set_target_properties(${qhull_SHAREDR} PROPERTIES
+     SOVERSION ${qhull_SOVERSION}
+-    VERSION ${qhull_VERSION})
++    VERSION ${qhull_VERSION}
++    OUTPUT_NAME "${qhull_SHAREDR}$<$<CONFIG:Debug>:d>")
+ 
+ if(UNIX)
+     target_link_libraries(${qhull_SHAREDR} m)
+@@ -397,16 +397,11 @@ endif(UNIX)
+ add_library(${qhull_SHARED} SHARED 
+         ${libqhull_SOURCES}
+         src/libqhull/qhull-exports.def)
+-        
+-if(qhull_SHARED MATCHES "libqhull")
+-   set(qhull_OUTPUT_NAME qhull)
+-   set_target_properties(${qhull_SHARED} PROPERTIES
+-        OUTPUT_NAME "${qhull_OUTPUT_NAME}" )
+-endif()
+ 
+ set_target_properties(${qhull_SHARED} PROPERTIES
+     SOVERSION ${qhull_SOVERSION}
+-    VERSION ${qhull_VERSION})
++    VERSION ${qhull_VERSION}
++    OUTPUT_NAME "qhull$<$<CONFIG:Debug>:_d>")
+ 
+ if(UNIX)
+     target_link_libraries(${qhull_SHARED} m)
+@@ -431,7 +426,8 @@ add_library(${qhull_SHAREDP} SHARED
+ set_target_properties(${qhull_SHAREDP} PROPERTIES
+     COMPILE_DEFINITIONS "qh_QHpointer"
+     SOVERSION ${qhull_SOVERSION}
+-    VERSION ${qhull_VERSION})
++    VERSION ${qhull_VERSION}
++    OUTPUT_NAME "${qhull_SHAREDP}$<$<CONFIG:Debug>:d>")
+ 
+ if(UNIX)
+     target_link_libraries(${qhull_SHAREDP} m)
+@@ -452,11 +448,13 @@ endif(UNIX)
+ 
+ add_library(${qhull_STATIC} STATIC ${libqhull_SOURCES})
+ set_target_properties(${qhull_STATIC} PROPERTIES
+-    VERSION ${qhull_VERSION})
++    VERSION ${qhull_VERSION}
++    OUTPUT_NAME "${qhull_STATIC}$<$<CONFIG:Debug>:_d>")
+ 
+ add_library(${qhull_STATICR} STATIC ${libqhullr_SOURCES})
+ set_target_properties(${qhull_STATICR} PROPERTIES
+-    VERSION ${qhull_VERSION})
++    VERSION ${qhull_VERSION}
++    OUTPUT_NAME "${qhull_STATICR}$<$<CONFIG:Debug>:d>")
+ 
+ if(UNIX)
+     target_link_libraries(${qhull_STATIC} m)
+@@ -471,6 +469,7 @@ endif(UNIX)
+ add_library(${qhull_CPP} STATIC ${libqhullcpp_SOURCES})
+ set_target_properties(${qhull_CPP} PROPERTIES
+     VERSION ${qhull_VERSION}
++    OUTPUT_NAME "${qhull_CPP}$<$<CONFIG:Debug>:_d>"
+     POSITION_INDEPENDENT_CODE "TRUE")
+ 
+ # ---------------------------------------
+@@ -561,10 +560,8 @@ target_link_libraries(user_eg2 ${qhull_STATICR})
+ 
+ set(user_eg3_SOURCES    src/user_eg3/user_eg3_r.cpp)
+ 
+-add_executable(user_eg3 ${user_eg3_SOURCES})
+ # qhull_STATICR must be last, otherwise qh_fprintf,etc. are not loaded from qhull_CPP
+ # user_eg3 may be linked to qhull_SHAREDR if user_eg3_DEFINES is added
+-target_link_libraries(user_eg3 ${qhull_CPP} ${qhull_STATICR})
+ 
+ # ---------------------------------------
+ # qhullp is qhull/unix.c linked to deprecated qh_QHpointer libqhull_p
+@@ -615,8 +612,6 @@ add_test(NAME user_eg
+    COMMAND sh -c "./user_eg")
+ add_test(NAME user_eg2
+    COMMAND sh -c "./user_eg2")
+-add_test(NAME user_eg3
```

Comment:
why eg3 what removed, but eg and eg2 kept?

Reply:
AFAIR `user_eg3` failed to build for some triplets.